content/docs/accessibility.md

```diff
@@ -1,32 +1,32 @@
 ---
 id: accessibility
-title: Accessibility
+title: Հասանելիություն
 permalink: docs/accessibility.html
 ---
 
-## Why Accessibility? {#why-accessibility}
+## Ինչու՞ հասանելիություն {#why-accessibility}
 
-Web accessibility (also referred to as [**a11y**](https://en.wiktionary.org/wiki/a11y)) is the design and creation of websites that can be used by everyone. Accessibility support is necessary to allow assistive technology to interpret web pages.
+Վեբ հասանելիությունը (հղվում է նաև [**a11y**](https://en.wiktionary.org/wiki/a11y)-ին) այնպիսի վեբ կայքի կառուցվածքն ու պատրաստումն է, որը կարող է օգտագործել յուրաքանչյուրը։ Հասանելիության ապահովումը անհրաժեշտ է, որպեսզի օգնական տեխնոլոգիաները հնարավորություն ունենան վեբ էջերը ինտերպրետացնելու։
 
-React fully supports building accessible websites, often by using standard HTML techniques.
+React֊ը լիովին հնարավորություն է տալիս սարքել հասանելի վեբ կայքեր, հաճախ օգտագործելով HTML֊ի ստանդարտ գործելակերպեր։
 
-## Standards and Guidelines {#standards-and-guidelines}
+## Ստանդարտներ և ուղեցույցներ {#standards-and-guidelines}
 
 ### WCAG {#wcag}
 
-The [Web Content Accessibility Guidelines](https://www.w3.org/WAI/intro/wcag) provides guidelines for creating accessible web sites.
+[Վեբ բովանդակության հասնաելիության ուղեցույցը](https://www.w3.org/WAI/intro/wcag) տրամադրում է ուղեցույց հասանելի վեբ կայքեր ստեղծելու համար։ 
 
-The following WCAG checklists provide an overview:
+WCAG֊ի հետևյալ ցուցակները տրամադրված են դիտարկման\`
 
-- [WCAG checklist from Wuhcag](https://www.wuhcag.com/wcag-checklist/)
-- [WCAG checklist from WebAIM](https://webaim.org/standards/wcag/checklist)
-- [Checklist from The A11Y Project](https://a11yproject.com/checklist.html)
+- [WCAG ցուցակ Wuhcag֊ից](https://www.wuhcag.com/wcag-checklist/)
+- [WCAG ցուցակ WebAIM֊ից](https://webaim.org/standards/wcag/checklist)
+- [Ցուցակ A11Y ծրագրից](https://a11yproject.com/checklist.html)
 
 ### WAI-ARIA {#wai-aria}
 
-The [Web Accessibility Initiative - Accessible Rich Internet Applications](https://www.w3.org/WAI/intro/aria) document contains techniques for building fully accessible JavaScript widgets.
+[Վեբ հասանելիության նախաձեռնություն - հարուստ հասանելիությամբ ինտերնետ հավելվածներ](https://www.w3.org/WAI/intro/aria) փաստաթուղթը պարունակում է լիովին հասանելի JavaScript widget-ներ կառուցելու գործելակերպեր։
 
-Note that all `aria-*` HTML attributes are fully supported in JSX. Whereas most DOM properties and attributes in React are camelCased, these attributes should be hyphen-cased (also known as kebab-case, lisp-case, etc) as they are in plain HTML:
+Ուշադրություն դարձրեք, որ բոլոր `aria-*` HTML ատրիբուտները լրիվ սպասարկվում են JSX֊ի կողմից։ Ի տարբերություն DOM֊ի մյուս ատրիբուտների և հատկություների, որոնք React֊ում ուղտագիր են, սրանք պետք է լինեն գծիկագիր(հայտնի է նաև ինչպես քաբաբագիր, լիսպագիր, և այլն), ինչպես մաքուր HTML ֊ում։
```

Comment:
ուղտագիր -> ուղտաԳիր
գծիկագիր -> գծիկա-գիր
քաբաբագիր -> քաբաբա-գիր
լիսպագիր -> լիսպա-գիր

as they are in plain HTML: -> ինչպես մաքուր HTML-ում են.